src/core/public/saved_objects/simple_saved_object.ts

```diff
@@ -51,17 +51,29 @@ export class SimpleSavedObject<T = unknown> {
   public migrationVersion: SavedObjectType<T>['migrationVersion'];
   public error: SavedObjectType<T>['error'];
   public references: SavedObjectType<T>['references'];
+  public updated_at: SavedObjectType<T>['updated_at'];
 
   constructor(
     private client: SavedObjectsClientContract,
-    { id, type, version, attributes, error, references, migrationVersion }: SavedObjectType<T>
+    {
+      id,
+      type,
+      version,
+      attributes,
+      error,
+      references,
+      migrationVersion,
+      // eslint-disable-next-line @typescript-eslint/naming-convention
+      updated_at,
```

Comment:
Why does it need to be `updated_at` instead of `updatedAt`?

Reply:
Thats the OpenSearch response. Don't think we can change that.